service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -487,6 +508,23 @@ protected void doStop(final Future<Void> stopFuture) {
         return result;
     }
 
+    /**
+     * Checks if any further connections for the given tenant are allowed.
+     *
+     * @param tenantConfig The tenant to check for.
+     * @return A succeeded future if further connections are allowed otherwise a failed future with failure message.
+     */
+    protected final Future<TenantObject> isFurtherConnectionsAllowed(final TenantObject tenantConfig) {
```

Comment:
can we please rename this to the same name used in `ResourceLimitChecks`, i.e. *isConnectionLimitExceeded*?